ResNet_constructor.ipynb

```diff
@@ -0,0 +1,1271 @@
+{
```

Comment:
<p>Try to reduce amount of prints / outputs to a minimum. There are a lot if things that reader don't need to see.</p><p> </p><p>And if you need to print something, make it properly: instead of just writing</p><p><code>model_name</code> </p><p>write something like</p><p><code>print(f'Model will be saved under name: {model_name}</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bestetc/May/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>